cli/tests/pcluster/templates/test_cluster_stack.py

```diff
@@ -240,3 +240,127 @@ def _get_cfn_init_file_content(template, resource, file):
     content_separator = content_join[0]
     content_elements = content_join[1]
     return content_separator.join(str(elem) for elem in content_elements)
+
+
+@freeze_time("2021-01-01T01:01:01")
+@pytest.mark.parametrize(
+    "config_file_name, expected_instance_tags, expected_volume_tags",
+    [
+        (
+            "slurm.full.yaml",
+            {},
+            {
+                "parallelcluster:cluster-name": "clustername",
+                "parallelcluster:node-type": "HeadNode",
+                "String": "String",
+                "two": "two22",
+            },
+        ),
+        (
+            "awsbatch.full.yaml",
+            {},
+            {
+                "parallelcluster:cluster-name": "clustername",
+                "parallelcluster:node-type": "HeadNode",
+                "String": "String",
+                "two": "two22",
+            },
+        ),
+        (
+            "scheduler_plugin.full.yaml",
+            {},
+            {
+                "parallelcluster:cluster-name": "clustername",
+                "parallelcluster:node-type": "HeadNode",
+                "String": "String",
+                "two": "two22",
+            },
+        ),
+    ],
+)
+def test_head_node_tags_from_launch_template(mocker, config_file_name, expected_instance_tags, expected_volume_tags):
```

Comment:
Great job adding this test?

Can you double check if all the tags listed here for the head node are present? https://docs.aws.amazon.com/parallelcluster/latest/ug/resources-tags-v3.html 

We can add a link to this documentation in the test too.

Reply:
Among the tags listed in [ParallelCluster Docs](https://docs.aws.amazon.com/parallelcluster/latest/ug/resources-tags-v3.html), only those prefixed with `parallelcluster:` are set via instance definition.

I noticed that in our docs we are missing the following tags and must be aligned:
1. `parallelcluster:attributes`
2. `parallelcluster:filesystem`
3. `parallelcluster:networking`

Reply:
We decided to not publish them in our official doc because they are internal tags and we're not sure we want to preserve them in future.